cli/core/Makefile

```diff
@@ -158,6 +158,20 @@ release-%: ## Create release for a platform
 
 	$(MAKE) build-tanzu-core-cli-oci-$(OS)-$(ARCH)
 
+## --------------------------------------
+## OS Packages
+## --------------------------------------
+.PHONY: apt-package
+apt-package: ## Build a debian package to use with APT
+	@if [ "$$(command -v docker)" == "" ]; then \
+		echo "Docker required to build apt package" ;\
+		exit 1 ;\
+	fi
+
+	@# Until we have things automated, hard-code the version which must be available as a release on github
+	@# This can be overridden by specifying the version of the command-line. E.g., VERSION=v0.25.1 make apt-package
+	docker run --rm -e VERSION=$${VERSION-v0.26.0} -v $(ROOT_DIR):$(ROOT_DIR) ubuntu $(MODULE_ROOT_DIR)/hack/apt/build_package.sh
```

Comment:
> question: would the automation involve building this target with as part of producing a github release?

Not sure yet to be honest. We need an SRP-compliant build which we currently (IIUC) manually upload to GitHub; that's why this script downloads the version from GitHub instead of building it itself.


> nit: is defaulting to v0.26.0 necessary? Even as a temporary step it seems rather arbitrary.

You mean to instead require the version to be specified explicitly when calling the script?  Sure, I can do that.  The script will eventually be called by an automation which will specify the version, so you're right we don't need a default.

Reply:
>that's why this script downloads the version from GitHub instead of building it itself
yeah the question is definitely tied to this point, but something to consider in the future. Seems like if some process is producing the binaries needed for the debian package build, it would be useful to extend said process to include the packaging work too.

>You mean to instead require the version to be specified explicitly when calling the script? 
Yep that's what mean.

Reply:
> something to consider in the future. Seems like if some process is producing the binaries needed for the debian package build, it would be useful to extend said process to include the packaging work too.

Agreed this would be a nice improvement.